2

```diff
@@ -0,0 +1,8 @@
+import lessonStates from './lessonStates'
```

Comment:
it's in a file simply called `2` at the root of the project. Kinda odd. The imports I understand 😉